src/helpers/createDtsExports.ts

```diff
@@ -76,16 +76,30 @@ export default classes;
       .filter(([classname]) => isValidVariable(classname));
 
     filteredClasses.forEach(([classname, originalClassname]) => {
-      const classRegexp = new RegExp(`${originalClassname}[\\s{]`, 'g');
-
-      const matchedLine = cssLines.findIndex((line) => classRegexp.test(line));
-      const matchedColumn =
-        matchedLine && cssLines[matchedLine].indexOf(originalClassname);
```

Comment:
Here, `matchedLine` could be -1. This shouldn't have happened, but the newly introduced regexp above didn't cover enough cases.